index.js

```diff
@@ -1,130 +1,128 @@
-var _ = require('lodash');
+'use strict';
+
 var Q = require('q');
 var npmKeyword = require('npm-keyword');
 var packageJson = require('package-json');
 var ghGot = require('gh-got');
 var EventEmitter = require('events').EventEmitter;
 
-var UPDATE_INTERVAL_IN_SECONDS = 60 * 60;
+var UPDATE_INTERVAL_IN_SECONDS = 60 * 60 + 1;
 
 
 function createComponentData(npm, gh) {
+  gh = gh || {
+    description: '',
+    owner: {
+      login: '',
+      html_url: '',
+    },
+    html_url: '',
+    stargazers_count: '',
+    watchers_count: ''
+  };
+
+  var description = npm.description && gh.description ? (
```

Comment:
Can we make this legible? It's very hard to see what is going on here. Use `if/else` when it's becoming very complex.